### PR TITLE
Align composer.json / parameters.yml.dist with ezplatform

### PR DESCRIPTION
**Pull request by lolautruche**
* Ensure `ezsystems/ezplatform-solr-search-engine` is present with associated params
* Switched to Mink selenium2 driver for Behat

### CODE REVIEW

**Comment by lolautruche** [comment]
Review ping @andrerom @bdunogier @glye @yannickroger @pspanja

**Comment by glye** [comment]
@ezrobot is speechless? :)

**Comment by andrerom** [comment]
> @ezrobot is speechless? :)

to many issues to fit in comment..

**Comment by andrerom** [comment]
Missing parts:
- behat.yml.dist *(selenium changes, file is here called behat.yml)*
- bin/.travis/install_composer_github_key.sh *(for github rate limit)*
- kernel ? *(solr change)*
- ezpublish/config/config.yml ? *(solr change)*
- ezpublish/config/ezpublish.yml.example
- .travis.yml

__But__, what is the goal to this change? casue we also lack the db config options, but just like this one Legacy Setup installer does not support these things. So I guess what you aim for here is it make it on pair with 5.4.5?

**Comment by lolautruche** [comment]
@andrerom this repo just doesn't work with master any more because of missing parameters, that's all.
What's missing here is mainly the kernel update I think.

**Comment by lolautruche** [comment]
Fixed

**Comment by lolautruche** [comment]
@ezrobot is fixed now :smile:

**Comment by andrerom** [comment]
Warmer :)

- bin/.travis/prepare_selenium2.sh
- .travis.yml
- ezpublish/config/routing_dev.yml

**Comment by lolautruche** [comment]
New attempt :-)

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    # run setup wizard to install Demo+Content and then test demo with content
   - INSTALL="demoContentNonUniqueDB" PROFILE="demo" TEST="content"
-    # test Legacy
-  - INSTALL="demoCleanNonUniqueDB" PROFILE="legacyAdmin" TEST="full"
-    # test REST
-  - INSTALL="demoCleanNonUniqueDB" PROFILE="rest" TEST="fullJson"
-  - INSTALL="demoCleanNonUniqueDB" PROFILE="rest" TEST="fullXml"
+  - PROFILE="demo" TEST="clean"
+  - PROFILE="rest" TEST="fullJson"
+  - PROFILE="rest" TEST="fullXml"
```
Comment (andrerom) [inline .travis.yml]: should not change this

File: .travis.yml [hunk 2]
```diff
@@ -23,21 +23,16 @@ branches:
...
+  - ./bin/.travis/prepare_selenium2.sh
 
 # setup requirements for running tests
 before_script:
   # Prepare eZ Publish (composer, permissions, assets, cache warmup)
   - ./bin/.travis/prepare_ezpublish.sh
-  # since setup wizard need to actually work to the other test suites to pass
-  # moved it to before_script so that the random timeout on package fetching
-  # don't fail travis
-  - php bin/behat --profile setupWizard --suite $INSTALL
```
Comment (andrerom) [inline .travis.yml]: This one and the change above should continue to use Setup Wizard for community imho, platform installer is not written/intended to work on 5.x install.

File: .travis.yml (hunk 1 above)
Comment (lolautruche) [inline .travis.yml]: You mean the whole diff block ?

**Comment by andrerom** [comment]
Besides comments, I guess we now will need to have a look at travis for what else is missing / or needs to be different here because of legacy.

File: .travis.yml (hunk 2 above)
Comment (lolautruche) [inline .travis.yml]: > This one and the change above

The block you commented above?

Comment (andrerom) [inline .travis.yml]: y

File: .travis.yml (hunk 1 above)
Comment (andrerom) [inline .travis.yml]: yes the env part, at least I think there is some logic around the use of 'INSTALL' for legacy, but it might be gone.

**Comment by andrerom** [comment]
Synced further, but think I'll revert the sahi changes actually, seems setup wizard fails becasue of Selenium and if it works with Sahi that is ok, But means we should skip platform ui tests, as they are written for Selenium now.

What was the motivation for the sync here?

**Comment by lolautruche** [comment]
> What was the motivation for the sync here?

I needed to work with this repo as a basis to fix some reported issues, and it appeared to be completely broken...

**Comment by andrerom** [comment]
Ready for review @lolautruche @emodric

**Comment by lolautruche** [comment]
Well, if it passes now, I'm obviously +1 :-)

**Comment by emodric** [comment]
Judging by the quick look, looks okay to me.

+1

**Pull request merged**